features/step_definitions/editorial_remarks.rb

```diff
@@ -1,3 +1,42 @@
+When(/^I visit the edition show page$/) do
+  begin_drafting_document type: "case_study", title: "New edition", previously_published: false
+  click_button "Save"
+  @edition = Edition.find_by!(title: "New edition")
+  visit admin_edition_path(@edition)
+end
+
+Then(/^the notes tab is not visible$/) do
+  expect(all(".nav-tabs").map(&:text)).not_to have_content "Notes"
+end
```

Comment:
Following on from our [Slack thread about Cucumber tests vs. MiniTest + Capybara](https://gds.slack.com/archives/C02L13S214K/p1659536841696059) – if we do implement the other tabs as feature specs, I think we'd get more mileage out of these step definitions if we made them more flexible.

For example, we could match any tab using a variable:

```suggestion
Then(/^the "([^"]*)" tab is not visible$/) do |tab_name|
  expect(all(".nav-tabs").map(&:text)).not_to have_content tab_name
end
```

Which we could use as:

```gherkin
Then the "Notes" tab is not visible
```